suave/builder/api/api_test.go

```diff
@@ -39,8 +39,16 @@ func (nullSessionManager) AddTransaction(sessionId string, tx *types.Transaction
 	return &SimulateTransactionResult{Logs: []*SimulatedLog{}}, nil
 }
 
-func (nullSessionManager) AddBundle(sessionId string, bundle Bundle) error {
-	return nil
+func (nullSessionManager) AddTransactions(sessionId string, txs types.Transactions) ([]*SimulateTransactionResult, error) {
```

Comment:
Can you call these functions in `TestAPI` test? It is to make sure that both server and client encode the same thing.

Reply:
Did for `AddTransactions` too